virt/support/virt-support-overview.adoc

```diff
@@ -6,12 +6,48 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You can collect data about your environment, monitor the health of your cluster and virtual machines (VMs), and troubleshoot {VirtProductName} resources with the following tools.
+You can request assistance from Red Hat Support, report bugs, collect data about your environment, and monitor the health of your cluster and virtual machines (VMs) with the following tools.
+
+[id="opening-support-tickets_{context}"]
+== Opening support tickets
+
+If you have encountered an issue that requires immediate assistance from Red Hat Support, you can submit a support case.
+
+If you want to report a bug, you can create a Jira issue directly.
+
+[id="submitting-support-case_{context}"]
+=== Submitting a support case
+
+To request support from Red Hat Support, follow xref:../../support/getting-support.adoc#support-submitting-a-case_getting-support[the instructions for submitting a support case].
+
+It is helpful to collect debugging data to provide the data with your request for support.
+
+[id="collecting-data-for-red-hat-support_{context}"]
+==== Collecting data for Red Hat Support
+
+You can gather debugging information by performing the following steps:
+
+xref:../../virt/support/virt-collecting-virt-data.adoc#virt-collecting-data-about-your-environment_virt-collecting-virt-data[Collecting data about your environment]::
+Configure Prometheus and Alertmanager and collect `must-gather` data for {product-title} and {VirtProductName}.
+
+xref:../../virt/support/virt-collecting-virt-data.adoc#virt-collecting-data-about-vms_virt-collecting-virt-data[Collecting data about VMs]::
+Collect `must-gather` data and memory dumps from VMs.
+
+// must-gather not supported for ROSA/OSD, per Dustin Row
+ifndef::openshift-rosa,openshift-dedicated[]
+xref:../../virt/support/virt-collecting-virt-data.adoc#virt-using-virt-must-gather_virt-collecting-virt-data[`must-gather` tool for {VirtProductName}]::
+Configure and use the `must-gather` tool.
+endif::openshift-rosa,openshift-dedicated[]
```

Comment:
The first two items in this section use `must-gather`, but the "configure" link is at the end. Is this something to do first/move to the top of the list?